package.json

```diff
@@ -4,9 +4,12 @@
   "private": true,
   "scripts": {
     "build": "tsc",
-    "start": "nodemon -e ts --exec \"(npm run lint -- --force && tsc && node build/server.js) || true\"",
+    "start": "npm run lint && npm run build && npm-run-all --parallel build:watch lint:watch serve:watch",
```

Comment:
we could take the `nodemon` out of the equation and simply re-run the `node build\server.js` command on each `onSuccess`. 
from `tsc-watch` docs
>The COMMAND will not run if the compilation failed.
>**The child process (COMMAND) will be terminated before creating a new one.**